### ISSUE TITLE
Html: jumpto navigation and jumpto search appear on many pages

### ISSUE THREAD

**Issue by gnosygnu** [gnosygnu/xowa#394]
This is related to the general CSS refresh in #380 . I'm seeing a lot more ```Jump to: search, navigation``` appear on pages. 

MediaWiki changed the generation of the jumpto recently: https://phabricator.wikimedia.org/source/Vector/browse/master/includes/templates/index.mustache . I don't know of an easy way to emulate this, short of doing a read of the file-system and seeing if they have the old jumpto css or the new one.

So, for now, I'm doing a halfway approach that should work for both old and new css.

Tested in the home-wiki with css from before #380 and from after #380 . Also tested with the soon-to-be-released 2019-03 dewiki.

**Comment by desb42** [comment]
Sorry to get a bit picky here. I have also been looking at the difference behaviour between xowa and mediawiki (this was touched on a little while ago in #264)

Looking at the phabricator code line 19-21

    <div id="jump-to-nav"></div>
    <a class="mw-jump-link" href="#mw-head">{{msg-jumptonavigation}}</a>
    <a class="mw-jump-link" href="#p-search">{{msg-jumptosearch}}</a>
and at Xow_portal_mgr.java code lines 206-210

    ( "\n    <div id=\"jump-to-nav\" class=\"mw-jump\">" // NOTE:class=mw-jump is for backward compatibility
    , "\n    <a class=\"mw-jump-link\" href=\"#mw-navigation\">~{jumpto}~{jumptonavigation}</a>"
    , "\n    <a class=\"mw-jump-link\" href=\"#p-search\">~{jumpto}~{jumptosearch}</a>"
    , "\n    </div>" // NOTE: </div> is for backward compatibility; current MW places right after jump-to-nav
    ), "jumpto", "jumptonavigation", "jumptosearch");

The closing \</div> is in a different place (which is acknowledged)
The href link for navigation in phibricator is `#mw-head` whereas in xowa it is `#mw-navigation`
xowa.gfs has a \<div> with id="mw-head" but no mw-navigation anywhere
I doubt that this is a significant issue

**Comment by gnosygnu** [comment]
Sorry, missed this comment.

> The closing </div> is in a different place (which is acknowledged)

Yup. I tried to make it like the Phabricator version, but it breaks old wiki css's (jumpto starts showing up). If this causes a problem in the future, I'll figure out a different approach.

> The href link for navigation in phibricator is #mw-head whereas in xowa it is #mw-navigation

Oops. Nice catch. Updated it just now.

> xowa.gfs has a <div> with id="mw-head" but no mw-navigation anywhere

Yeah, the mw-head is from this line: https://phabricator.wikimedia.org/source/Vector/browse/master/includes/VectorTemplate.php$102

It looks like mw-navigation is otherwise deprecated.

Let me know if anything else. Otherwise, will close out in a few days. Thansk!
